stdlib/email/src/main/ballerina/src/email/Module.md

```diff
@@ -0,0 +1,83 @@
+## Module overview
+
+This module contains functions to perform email operations such as sending and reading emails using SMTP, POP3 and IMAP4 protocols.
+
+## Sample
+
+The sample given below uses the functions in the module to send an email with SMTP protocol.
+
+```ballerina
+import ballerina/io;
+import ballerina/email;
+
+email:SmtpConfig smtpConfig = {
+    connConfig: {
```

Comment:
There is no `connConfig`. remove it from the docs

Reply:
Removed and replaced with correct content.